DEBUG 2025-04-10 14:13:49.495 UTC: 	 CRAB Client version: v3.250318
DEBUG 2025-04-10 14:13:49.495 UTC: 	 Running on: Linux lxplus965.cern.ch 5.14.0-503.26.1.el9_5.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Feb 10 05:23:56 EST 2025 x86_64 x86_64 x86_64 GNU/Linux
DEBUG 2025-04-10 14:13:49.495 UTC: 	 Not using Singularity
DEBUG 2025-04-10 14:13:49.495 UTC: 	 Operating System: Red Hat Enterprise Linux 9.5 (Plow)
DEBUG 2025-04-10 14:13:49.495 UTC: 	 Executing command: 'submit'
DEBUG 2025-04-10 14:13:49.495 UTC: 	 Found CRAB cache file /afs/cern.ch/user/v/vdamante/.crab3
INFO 2025-04-10 14:13:49.497 UTC: 	 Will use CRAB configuration file crab3_unpackAndMatchCSCGEM.py
DEBUG 2025-04-10 14:13:49.497 UTC: 	 Loading CRAB configuration file.
ERROR 2025-04-10 14:13:49.501 UTC: 	 The 'maxJobRuntimeMin' parameter is not compatible with the 'Automatic' splitting mode (default).
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2025-04-10 14:13:49.501 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 157, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 135, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])  # the crab command to be executed
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/submit.py", line 37, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/SubCommand.py", line 326, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/SubCommand.py", line 95, in loadConfig
    raise ConfigurationException(configmsg)
CRABClient.ClientExceptions.ConfigurationException: The 'maxJobRuntimeMin' parameter is not compatible with the 'Automatic' splitting mode (default).
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2025-04-10 14:13:49.504 UTC: 	 Log file is /afs/cern.ch/work/v/vdamante/CMSSW_14_2_0_pre1/src/GEMCSCTriggerTest/CSCSlopeFinder/test/crab.log
DEBUG 2025-04-10 14:14:13.642 UTC: 	 CRAB Client version: v3.250318
DEBUG 2025-04-10 14:14:13.643 UTC: 	 Running on: Linux lxplus965.cern.ch 5.14.0-503.26.1.el9_5.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Feb 10 05:23:56 EST 2025 x86_64 x86_64 x86_64 GNU/Linux
DEBUG 2025-04-10 14:14:13.643 UTC: 	 Not using Singularity
DEBUG 2025-04-10 14:14:13.643 UTC: 	 Operating System: Red Hat Enterprise Linux 9.5 (Plow)
DEBUG 2025-04-10 14:14:13.643 UTC: 	 Executing command: 'submit'
DEBUG 2025-04-10 14:14:13.643 UTC: 	 Found CRAB cache file /afs/cern.ch/user/v/vdamante/.crab3
INFO 2025-04-10 14:14:13.646 UTC: 	 Will use CRAB configuration file crab3_unpackAndMatchCSCGEM.py
DEBUG 2025-04-10 14:14:13.646 UTC: 	 Loading CRAB configuration file.
DEBUG 2025-04-10 14:14:13.651 UTC: 	 Will use CRAB plugin ANALYSIS (i.e. will run an analysis job type).
ERROR 2025-04-10 14:14:13.651 UTC: 	 Working area '/afs/cern.ch/work/v/vdamante/CMSSW_14_2_0_pre1/src/GEMCSCTriggerTest/CSCSlopeFinder/test/2024_ZMu_31Mar2025/crab_2024_ZMu_31Mar2025' already exists 
Please change the requestName in the config file
ERROR 2025-04-10 14:14:13.651 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 157, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 135, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])  # the crab command to be executed
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/submit.py", line 37, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/SubCommand.py", line 328, in __init__
    self.requestarea, self.requestname, self.logfile = createWorkArea(
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/ClientUtilities.py", line 392, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
CRABClient.ClientExceptions.ConfigException: Working area '/afs/cern.ch/work/v/vdamante/CMSSW_14_2_0_pre1/src/GEMCSCTriggerTest/CSCSlopeFinder/test/2024_ZMu_31Mar2025/crab_2024_ZMu_31Mar2025' already exists 
Please change the requestName in the config file
INFO 2025-04-10 14:14:13.654 UTC: 	 Log file is /afs/cern.ch/work/v/vdamante/CMSSW_14_2_0_pre1/src/GEMCSCTriggerTest/CSCSlopeFinder/test/crab.log
DEBUG 2025-04-10 14:20:47.639 UTC: 	 CRAB Client version: v3.250318
DEBUG 2025-04-10 14:20:47.639 UTC: 	 Running on: Linux lxplus965.cern.ch 5.14.0-503.26.1.el9_5.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Feb 10 05:23:56 EST 2025 x86_64 x86_64 x86_64 GNU/Linux
DEBUG 2025-04-10 14:20:47.639 UTC: 	 Not using Singularity
DEBUG 2025-04-10 14:20:47.639 UTC: 	 Operating System: Red Hat Enterprise Linux 9.5 (Plow)
DEBUG 2025-04-10 14:20:47.639 UTC: 	 Executing command: 'submit'
DEBUG 2025-04-10 14:20:47.640 UTC: 	 Found CRAB cache file /afs/cern.ch/user/v/vdamante/.crab3
INFO 2025-04-10 14:20:47.641 UTC: 	 Will use CRAB configuration file crab3_unpackAndMatchCSCGEM.py
DEBUG 2025-04-10 14:20:47.642 UTC: 	 Loading CRAB configuration file.
ERROR 2025-04-10 14:20:47.646 UTC: 	 Invalid CRAB configuration: In case of Automatic splitting, the Data.unitsPerJob parameter must be in the [180, 2700] minutes range. You asked for 10 minutes.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2025-04-10 14:20:47.646 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 157, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/bin/crab.py", line 135, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])  # the crab command to be executed
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/submit.py", line 37, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/SubCommand.py", line 326, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.250318.00/lib/CRABClient/Commands/SubCommand.py", line 95, in loadConfig
    raise ConfigurationException(configmsg)
CRABClient.ClientExceptions.ConfigurationException: Invalid CRAB configuration: In case of Automatic splitting, the Data.unitsPerJob parameter must be in the [180, 2700] minutes range. You asked for 10 minutes.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
INFO 2025-04-10 14:20:47.648 UTC: 	 Log file is /afs/cern.ch/work/v/vdamante/CMSSW_14_2_0_pre1/src/GEMCSCTriggerTest/CSCSlopeFinder/test/crab.log
